### DISCUSSION TITLE
github-copilot-for-business-faq

### DISCUSSION THREAD

**Discussion by giscus** [WorkSafeBC-Common-Engineering/Common.Engineering.Discussion#21] [Technical Blog]
# github-copilot-for-business-faq

https://wsbctechnicalblog.github.io/github-copilot-for-business-faq.html

**Comment by MerlinDarkstar** [comment]
Can I use two different browsers that are connected to each of my two different GitHub accounts, so I can continue to do work on my open source projects with GitHub (public) and then have my business work (using GitHub for Business) on the other browser - and just switch browser default within VS?

**Reply by AndreasNVI** [reply]
Yes, you can.
1. Log into your WSBC GitHub for Business account in a regular browser session.
2. Log into your individual/personal GitHub account in an InPrivate browser session.
You cannot do this the other way around though - you cannot log into your WSBC GitHub account in an InPrivate browser session (doesn't meet security requirements)

**Reply by wpschaub** [reply]
I have updated our FAQ post with great guidance from Cσʅιɳ DҽɱႦσʋʂƙყ.

**Reply by AndreasNVI** [reply]
Just tried out the switch accounts and it works well within the browsers. I also found with the new VS 2022 (17.8.0), if you have one GitHub account and you select to add another GitHub account, it will prompt you with the switch account view in the browser to select the account you want, and once selected that becomes the sole GitHub account. So, you cannot have two GitHub accounts available in VS, but it is easier now to switch.

**Reply by AndreasNVI** [reply]
Update to above, it does switch to the other GitHub account, but it asks to re-authenticate. It works for the browser, but not in VS.
